frontend/src/components/header/header.graphql

```diff
@@ -1,23 +1,45 @@
-fragment Link on MenuLink {
-  title(language: $language)
-  href(language: $language)
-
-  page {
-    slug(language: $language)
-  }
-}
-
-query Header($code: String!, $language: String!) {
+query Header($code: String!) {
   conference(code: $code) {
     id
-    conferenceMenu: menu(identifier: "conference-nav") {
+    conferenceMenuEn: menu(identifier: "conference-nav") {
       links {
-        ...Link
+        title(language: "en")
+        href(language: "en")
+
+        page {
+          slug(language: "en")
+        }
       }
     }
-    programMenu: menu(identifier: "program-nav") {
+    programMenuEn: menu(identifier: "program-nav") {
       links {
-        ...Link
+        title(language: "en")
+        href(language: "en")
+
+        page {
+          slug(language: "en")
+        }
+      }
+    }
+
+    conferenceMenuIt: menu(identifier: "conference-nav") {
+      links {
+        title(language: "it")
+        href(language: "it")
+
+        page {
+          slug(language: "it")
+        }
+      }
+    }
+    programMenuIt: menu(identifier: "program-nav") {
+      links {
+        title(language: "it")
+        href(language: "it")
+
+        page {
+          slug(language: "it")
```

Comment:
do you think in future we should move the language to be a header? We can chat about this on slack, just curious of what you think :D

Reply:
having the language in the header means that apollo won't be able to cache based on the language 🤔

Reply:
yup, but maybe there's a way to prevent that, that's something we could try in future, mostly because I'm curios to see if it is possible 😊

Reply:
This is an issue I found a few days ago of someone that has the language in the header: https://github.com/apollographql/apollo-client/issues/1420 and ended up having to send the language as variable to fix the cache